docs/source/upgrade-guide/index.md

```diff
@@ -16,18 +16,40 @@ Volto uses Semantic Versioning.
 For more information see {doc}`../contributing/version-policy`.
 
 ```{note}
-There are times when updating the Volto boilerplate (the one generated by `@plone/generator-volto`) is enough to fulfill all the changes.
-If you have not heavily modified it, moving things around and copying over your dependencies might do when dealing with upgrades.
-We keep the generator up to date and in sync with the current Volto release.
-When it runs, the generator will tell you whether it is outdated.
-The generator will also "update" your project with the latest changes, and propose to merge the changes.
-Thus it is safe to run it on top of your project and answer the prompts.
+[Cookieplone](https://github.com/plone/cookieplone) is the official boilerplate generator for Plone projects.
+We keep Cookieplone up to date and in sync with the current Volto release.
+We encourage you to keep all your code inside your project add-on(s).
+If you do so, once you want to upgrade your project, you can generate a new project using Cookieplone with the same name as your old one, and copy over the add-ons to the new boilerplate.
+Even if you have modified the boilerplate, moving things around and copying over your dependencies is better and quicker than dealing with following the upgrade steps.
 ```
 
 (volto-upgrade-guide-18.x.x)=
 
 ## Upgrading to Volto 18.x.x
 
+(upgrade-18-cookieplone-label)=
+
+### Cookieplone is now the recommended project and add-on generator for Volto 18
+
+```{versionadded} Volto 18.0.0-alpha.43
+```
+
+[Cookieplone](https://github.com/plone/cookieplone) is now the recommended way to develop Volto projects, using it as a boilerplate generator.
+Cookieplone uses the frontend code installed using `pnpm` instead of `yarn`.
+This affects the way that the Plone Release Team generates the Sponsored OSS Docker images, since they must be compatible with the `pnpm` setup.
+
+Since Volto `18.0.0-alpha.43`, the Docker image [`plone-frontend`](https://hub.docker.com/r/plone/plone-frontend) uses `pnpm`.
+
+For developers that won't migrate their boilerplate and code to the new setup with `pnpm` instead of `yarn`, the Release Team will generate a new Docker image named `plone-frontend:18-yarn` for the Volto 18 series of alpha releases.
```

Comment:
```suggestion
For developers that won't migrate their projects to the new setup with `pnpm` instead of `yarn`, the Release Team will generate a new Docker image named `plone-frontend:18-yarn` for the Volto 18 series of alpha releases.
```